content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -121,6 +121,7 @@ <h3>kubectl basics</h3>
                 <a id="deploy-an-app"></a>
                 <h3>Deploy an app</h3>
                 <p>Let’s deploy our first app on Kubernetes with the <code>kubectl create deployment</code> command. We need to provide the deployment name and app image location (include the full repository url for images hosted outside Docker Hub).</p>
+                <div class="alert alert-info note callout" role="alert"><strong>Note:</strong> This only works on certain CPU architectures and may not work on your machine.</div>
```

Comment:
Thanks

Can we use a note shortcode here? See https://kubernetes.io/docs/contribute/style/hugo-shortcodes/

Reply:
Yeah. Those docs don't explicitly mention a note shortcode, but I do see `{{< note >}}` used in some places. So I guess that is a shortcode.

I now see a note at the top of the page, so maybe this note is not needed.

The note at the top is easy to disregard. It is not until after the `kubectl create deployment` and `kubectl get deployments` that you find out that the image will not work.

Perhaps that is the best place to mention something like "If you see 0/1 ready then you will need to use a different image that was works on your machine, a machine with an AMD64 CPU, or a driver that can emulate AMD64"

Most of the tutorial still works if you use the `registry.k8s.io/e2e-test-images/agnhost:2.39` image from the Hello Minikube page. That image is built for multiple architectures. It is a shame to tell someone at the start of the tutorial that "it might not work on your machine". I think they would prefer to focus on getting through most of the tutorial content rather than being distracted by this issue. The only part I've found that doesn't work with that image is the `cat server.js` command in Viewing Pods and Nodes.

The note at the top doesn't really explain what to do to make it work. I'm assuming that installing Docker Desktop is required. Since I can legally get Docker Desktop for free, I'll try it. I don't know if there is some other way to get the appropriate driver.  I don't really even know what "driver" means in this context.

Reply:
After much investigating I figured out how to run the tutorial's image successfully on an Apple chip. I update the note at the top of the page to make it clear what to do.

It can still be easily be overlooked, because it requires the reader to recognize the chip architecture name/code mentioned and know that it doesn't match their machine.